cloudbuild.yaml

```diff
@@ -25,7 +25,15 @@ steps:
         "gcr.io/cloud-devrel-public-resources/storage-testbench:$TAG_NAME",
         "gcr.io/cloud-devrel-public-resources/storage-testbench:latest",
       ]
+  - name: gcr.io/cloud-builders/docker
```

Comment:
I'm not 100% sure but you may have to add `waitFor` since we are going to have multiple tags for this image now. For example: [python-docker-image](https://github.com/bajajneha27/testing-infra-docker/blob/0edfe0adc4f4e4e3ee610134c1ab368edc8bd396/python/cloudbuild.yaml#L56)

Reply:
So far this hasn't seemed to be necessary but I'll keep an eye on it with the next release, thank you Neha!